tests/app/core/models/session-model.spec.ts

```diff
@@ -55,6 +49,47 @@ describe('Session Model', () => {
         });
     });
 
+    it('should throw error on session data with invalid open phases', () => {
+      of({ ...validSessionData, openPhases: ['unknownPhase'] })
+        .pipe(assertSessionDataIntegrity())
+        .subscribe({
+          next: () => fail(),
+          error: (err) => expect(err).toBeInstanceOf(Error),
```

Comment:
I was thinking in the case where we modify error messages it might be less work to change the test case for particular strings but if we are importing  as constants I guess thats okay 👍